vpdq/cpp/regtest.py

```diff
@@ -72,91 +74,114 @@ def get_argparse() -> argparse.ArgumentParser:
     return ap
 
 
-def dir_path(string):
-    if os.path.isdir(string):
-        return string
-    else:
-        raise argparse.ArgumentTypeError(f"readable_dir: {string} is not a valid path")
+def main():
+    hashVideoExecutable = EXEC_DIR / "vpdq-hash-video"
+    matchHashesExecutable = EXEC_DIR / "match-hashes-byline"
 
+    if not hashVideoExecutable.exists() or not matchHashesExecutable.exists():
+        print(
+            "Error: Hashing executable/s not found. Build vpdq before running regtest."
+        )
+        sys.exit(1)
```

Comment:
(ignorable): You can turn these into a validator on argparse and raise ArgumentError to get slightly more consistent behavior: https://docs.python.org/3/library/argparse.html#type

Reply:
These aren't CLI parameters. Do you mean just use common function for every check for file/folder validity?